content/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/troubleshooting-required-status-checks.md

```diff
@@ -59,6 +59,8 @@ If there is a conflict between the test merge commit and head commit, the checks
 
 If, however, a job within a workflow is skipped due to a conditional, it will report its status as "Success". For more information, see "[AUTOTITLE](/actions/using-jobs/using-conditions-to-control-job-execution)."
 
+A job that is skipped because any dependent job listed with `needs` has failed, will not be reported as a failed check. A pull request that requires this check will be allowed to be merged.  To use a required check on a job that depends on other jobs, use the `always()` in addition to `needs`, see "[AUTOTITLE](/actions/using-jobs/using-jobs-in-a-workflow#defining-prerequisite-jobs)."
+
```

Comment:
Thanks for adding this additional information 💖 

I'm going to make some small updates then this should be okay to merge.

```suggestion
When a job fails, any jobs that depend on the failed job are skipped and do not report a failure. A pull request that requires the check may not be blocked. To use a required check on a job that depends on other jobs, use the `always()` conditional expression in addition to `needs`, see "[AUTOTITLE](/actions/using-jobs/using-jobs-in-a-workflow#defining-prerequisite-jobs)."
```